client/app/bundles/course/assessment/submission/pages/SubmissionEditIndex/index.jsx

```diff
@@ -106,6 +106,17 @@ class VisibleSubmissionEditIndex extends Component {
     return numIncorrect === 0;
   }
 
+  allowPartialSubmission() {
+    const { assessment, explanations, questions } = this.props;
+    const numCorrect = Object.keys(explanations).filter(
```

Comment:
I'm wondering if we should just remove all this logic and allow submission even if nothing has been answered correctly. This will be consistent with the behaviour of manually graded assessments.

So whether this is allowed or not just depends on the boolean in the assessment.